.github/wiki/Getting-Started-Storyboard.md

```diff
@@ -243,3 +243,26 @@ At a high level we are loading the view controller for testing (similar to befor
 <details>
```

Comment:
Now that these are in the Jazzy docs.  I'm not sure we still need the wiki files anymore.  We could delete this file, right?

Reply:
Not as part of this GET YOUR OWN PR FOR DOCUMENTATION STUFF